data/human/outfits.txt

```diff
@@ -391,6 +391,16 @@ outfit "Nerve Gas"
 	"unplunderable" 1
 	description "This is a highly illegal weapon for hand-to-hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."
 
+outfit "Drugs"
+	plural "Drugs"
+	category "Hand to Hand"
+	cost 10000
+	thumbnail "outfit/Drugs"
+	"capture attack"  2.2
+	"capture defense" 1.3
+	"illegal" 800000
+	"unplunderable" 1
+	description " Although they decrease your crew's jugment and are illegal, drugs can increase your crew's fighting spirit and will numb them to a certain amount of pain."
```

Comment:
Surely a drug like this would be severely addictive and destructive - look at what happened to Nazis who consumed Panzerschokoloade. Would propose adding consequence events rather than letting First Last get off scot-free.

Reply:
Panzerschokolade was just meth iirc, and not just Germans did it - the Allies also did distribute meth to soldiers.

But seconding you on consequences of some sort. Maybe the crew experiences severe flashbacks and PTSD or end up completely divorced from reality? The latter happened to a USA soldier who volunteered for combat drug experiments around 1970...